src/SerilogTracing/Instrumentation/ActivityInstrumentation.cs

```diff
@@ -248,6 +254,199 @@ class TextException(
         public override string ToString() => toString ?? "No information available.";
     }
 
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="configureReplacement"></param>
+    /// <param name="postSamplingFilter"></param>
+    /// <param name="inheritTags"></param>
+    /// <param name="inheritParent"></param>
+    /// <param name="inheritFlags"></param>
+    /// <param name="inheritBaggage"></param>
+    /// <returns></returns>
+    public static void StartReplacementActivity(
+        Func<Activity?, bool> postSamplingFilter,
+        Action<Activity> configureReplacement,
+        bool inheritTags = true,
+        bool inheritParent = true,
+        bool inheritFlags = true,
+        bool inheritBaggage = true
+    ) {
+        var replace = Activity.Current;
```

Comment:
We might want to pass the activity to replace in to this function, but it only works if the activity is `Activity.Current`, since we mess with `Activity.Current` along the way, so opted to grab the value here.